openwisp_utils/admin_theme/admin.py

```diff
@@ -38,6 +55,11 @@ def get_urls(self):
                 self.admin_view(autocomplete_view.as_view(admin_site=self)),
                 name='ow-auto-filter',
             ),
+            path(
+                'ow-info/',
```

Comment:
Why not something more readable? Eg:

```suggestion
                'openwisp-system-info/',
```